### ISSUE TITLE
PHP Protobuf when i used new Google\Protobuf\Any() in php-fpm will be 502

### ISSUE THREAD

**Issue by xxlixin1993** [google/protobuf#4140]
php version is 7.0.18, protobuf extension is v3.5.1. It always look like two failed twice successfully when i use nginx + php-fpm try 4 times. Then i open core dump find 
```
[0x7f6f78213290] Google\Protobuf\Any->__construct() [internal function]
[0x7f6f78213030] (main) /home/lixin/work/test/protobuf/test/client.php:15 
```

```
syntax = "proto3";

package message;

import "google/protobuf/any.proto";

enum MsgPriority {
    PRIORITY0 = 0;
    PRIORITY1 = 1;
    PRIORITY2 = 2;
    PRIORITY3 = 3;
    PRIORITY4 = 4;
    PRIORITY5 = 5;
    PRIORITY6 = 6;
    PRIORITY7 = 7;
    PRIORITY8 = 8;
    PRIORITY9 = 9;
}

message Message {
    reserved 4 to 15; // 保留序号

    string MessageId = 1; // 消息ID
    MsgPriority priority = 2; // 消息优先级
    map<string, string> options = 100; // 消息选项
    google.protobuf.Any Body = 3; // 消息具体内容
}
```
```
syntax = "proto3";
package protocol;

message SmsTask {
    reserved 5 to 10;
    string templateId = 1; 
    string phone = 2; 
    string content = 3; 
    int64 time = 4; 
}
```
```
<?php
require('../GPBMetadata/SmsTask.php');
require('../Protocol/SmsTask.php');
require('../GPBMetadata/Common.php');
require('../Message/Message.php');
require('../Message/MsgPriority.php');

$pbSmsTask = new \Protocol\SmsTask();

$pbSmsTask->setTime(123);
$pbSmsTask->setContent('abc');
$pbSmsTask->setPhone(13259775719);
$pbSmsTask->setTemplateId(1);

$any = new Google\Protobuf\Any();

$any->pack($pbSmsTask);
$message = new \Message\Message();
$message->setBody($any);
$str = $message->serializeToString();

var_dump($str);exit; 
```

**Comment by TeBoring** [comment]
Could you try the fix in #4139?

**Comment by xxlixin1993** [comment]
I'm so sorry to answer you so long. I test the tag 3.5.1.1 that is success.I think its fixed. THX
